scout/index.md

```diff
@@ -80,11 +78,31 @@ $ curl -fsSL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh
 $ sh install-scout.sh
 ```
 
-Always examine scripts downloaded from the internet before running them locally. Before installing, make yourself familiar with potential risks and limitations of the convenience script:
-
-> **Tip**
+> **Note**
 >
-> If you want to install the plugin manually, you can find full instructions in the [plugin's repository](https://github.com/docker/scout-cli).
-{: .tip }
+> Always examine scripts downloaded from the internet before running them locally.
+> Before installing, make yourself familiar with potential risks and limitations
+> of the convenience script
+
+If you want to install the plugin manually, you can find full instructions in
+the [plugin's repository](https://github.com/docker/scout-cli).
+
+The plugin is also available as [a container image](https://hub.docker.com/r/docker/scout-cli)
+and as [a GitHub action](https://github.com/docker/scout-action).
+
+## Docker Scout Dashboard
+
+The [Docker Scout Dashboard](https://scout.docker.com){: target="\_blank" rel="noopener" }
+helps you share the analysis and security status of images in
+an organization with your team. You can also [use the dashboard to enable analysis of multiple repositories at once](./dashboard.md#repository-settings).
+
+For more information, read the [Docker Scout Dashboard guide](./dashboard.md).
+
+## JFrog Artifactory integration
+
+Users of JFrog Artifactory, or JFrog Container Registry, can integrate Docker
+Scout to enable automatic analysis of images locally and remotely. For more
+information, see [Artifactory integration](./artifactory.md).
+
 
-The plugin is also available as [a container image](https://hub.docker.com/r/docker/scout-cli) and as [a GitHub action](https://github.com/docker/scout-action)
\ No newline at end of file
+<div style="position: relative; padding-bottom: 64.86486486486486%; height: 0;"><iframe src="https://www.loom.com/embed/a6fb14ede0a94d0d984edf6cf16604e0?sid=ba34f694-32a6-4b74-b3f8-9cc6b80ef66f" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
```

Comment:
Could you add a description about the video?

Reply:
Done, this one doesn't completely fit here, but the team want to host it somewhere and when the onboarding guide is finished, there will be a better place to put it.